examples/dev/trace-analytics-sample-app/docker-compose.yml

```diff
@@ -78,7 +78,7 @@ services:
 
   opensearch:
     container_name: node-0.example.com
-    image: opensearchproject/opensearch:1.0.0
+    image: opensearchproject/opensearch:latest
```

Comment:
We should use the latest known major version so that our samples are not broken by changes to OpenSearch. Since OpenSearch is using semver, we can be confident in any 2.x version. OpenSearch supports a `2` label that is the current latest of the 2.x series. Please use `2` instead of `latest`.